src/Core/Language.cs

```diff
@@ -56,6 +56,11 @@ public sealed class Language : IEquatable<Language>
         public static Language GetLanguageFromLanguageKey(string languageKey) =>
             KnownLanguages.FirstOrDefault(l => languageKey.Equals(l.ServerLanguage.Key, System.StringComparison.OrdinalIgnoreCase));
 
+        /// <summary>
+        /// Tries to get the language for the specified repository key.
+        /// </summary>
+        public static bool GetLanguageFromRepositoryKey(string repoKey, out Language language) => repoKeyToLanguage.TryGetValue(repoKey, out language);
```

Comment:
It seems a bit odd that the `Get` methods have similar names (`GetLanguageFromLanguageKey`, `GetLanguageFromRepositoryKey`), but have different signatures.

Your new method is really a `TryGet`. Does it need to be?
If it does, the test needs to check the return value as well as the `out` value.

Reply:
It does not need to be, not sure why I have made it that way